pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -2402,6 +2402,12 @@ struct pjsua_msg_data
      */
     pj_str_t    target_uri;
 
+    /**
+     * Optional local URI (i.e. From header). If NULL, the
+     * the contact is used for the From header.
```

Comment:
Need to add doc which APIs will this apply to, i.e.: `This field is currently only used by ....`?
Is it only applicable to pjsua_call_make_call() only?

Reply:
It was. Latest change adds `pjsua_im_send()`.